### ISSUE TITLE
too many initializers

### ISSUE THREAD

**Issue by drc85** [evilgeniuslabs/tree-v2#20]
hi first i want to say great tutorial.. im a total beginner and i managed to solder and put everything together like on the website... i got a wifi shield its also very nice its safes a lot of work and space...
i installed all libary and the esp flasher they all work...

unfortunaly i cant get ur code running and up on the chip... but i tried a few codes and got the led running so i know for sure its wired up ok and i made no mistakes....

also how do i upload  the files to the webserver im not really sure im totally new to this ... took me a hole day to get the test stripe running :D

here is my error code, it is inside the map.h file....
probably because i only have 100 led
also im not really sure how to map my led since its totaly diffrent and unevven LOL
is there a simple way i just can play a few simple patterns for chritmas at least?
and controll them over the web app?

here is the code

 `Arduino: 1.8.5 (Windows 10), Board: "WeMos D1 R2 & mini, 160 MHz, 921600, 4M (3M SPIFFS)"

In file included from C:\Users\MCLOW\Desktop\tree-v2\tree-v2.ino:21:0:

C:\Users\MCLOW\Documents\Arduino\libraries\FastLED-master/FastLED.h:17:21: note: #pragma message: FastLED version 3.001.006

 #    pragma message "FastLED version 3.001.006"

                     ^

In file included from C:\Users\MCLOW\Documents\Arduino\libraries\FastLED-master/FastLED.h:68:0,

                 from C:\Users\MCLOW\Desktop\tree-v2\tree-v2.ino:21:

C:\Users\MCLOW\Documents\Arduino\libraries\FastLED-master/fastspi.h:110:23: note: #pragma message: No hardware SPI pins defined.  All SPI access will default to bitbanged output

 #      pragma message "No hardware SPI pins defined.  All SPI access will default to bitbanged output"

                       ^

In file included from C:\Users\MCLOW\Desktop\tree-v2\tree-v2.ino:74:0:

Map.h:21: error: too many initializers for 'const uint8_t [97] {aka const unsigned char [97]}'

 };

 ^

In file included from C:\Users\MCLOW\Desktop\tree-v2\tree-v2.ino:74:0:

Map.h:25: error: too many initializers for 'const uint8_t [97] {aka const unsigned char [97]}'

 };

 ^

In file included from C:\Users\MCLOW\Desktop\tree-v2\tree-v2.ino:74:0:

Map.h:29: error: too many initializers for 'const uint8_t [97] {aka const unsigned char [97]}'

 };

 ^

Map.h:89: error: too many initializers for 'const uint8_t [97] {aka const unsigned char [97]}'

 };

 ^

In file included from C:\Users\MCLOW\Desktop\tree-v2\tree-v2.ino:74:0:

Map.h:108: error: too many initializers for 'const uint8_t [97] {aka const unsigned char [97]}'

 };

 ^

In file included from C:\Users\MCLOW\Desktop\tree-v2\tree-v2.ino:74:0:

Map.h:112: error: too many initializers for 'const uint8_t [97] {aka const unsigned char [97]}'

 };

 ^

In file included from C:\Users\MCLOW\Desktop\tree-v2\tree-v2.ino:74:0:

Map.h:116: error: too many initializers for 'const uint8_t [97] {aka const unsigned char [97]}'

 };

 ^

exit status 1
too many initializers for 'const uint8_t [97] {aka const unsigned char [97]}'

Dieser Bericht wäre detaillierter, wenn die Option
"Ausführliche Ausgabe während der Kompilierung"
in Datei -> Voreinstellungen aktiviert wäre.
`

**Comment by jasoncoon** [comment]
Yeah, you'll need to rework the dimensional map arrays in Map.h to fit your LED count and layout.  It helps to just start with the basic [ESP8266 FastLED WebServer](https://github.com/jasoncoon/esp8266-fastled-webserver) sketch, get that working, then start mapping your LEDs in three dimensional space, if needed.
